src/app.js

```diff
@@ -1,15 +1,22 @@
+import {inject} from 'aurelia-framework';
+import {I18N} from 'aurelia-i18n';
+import {EventAggregator} from 'aurelia-event-aggregator';
 import {parseJSON} from './resources/jsonparser';
+import {Field} from './resources/elements/abstract/field';
 import {schema, fieldsToShow} from './schemas/index';
 import YAML from 'yamljs';
 import $ from 'jquery';
 
+@inject(I18N, EventAggregator)
 export class App {
-  constructor() {
+  constructor(i18n, ea) {
+    Field.internationalizer = i18n;
+    Field.eventAggregator = ea;
     // Allow access from browser console
     window.$oai = this;
 
     try {
-      this.forms = parseJSON('swagger', JSON.parse(JSON.stringify(schema)));
+      this.forms = parseJSON('form', JSON.parse(JSON.stringify(schema)));
       if (window.localStorage['openapi-v2-design']) {
         this.forms.setValue(JSON.parse(window.localStorage['openapi-v2-design']));
```

Comment:
Lets use a named variable here, since there are so many levels of nesting. E.g.

```js
// Get previous OpenAPI v2 design from local storage
const previousDesign = JSON.parse(window.localStorage['openapi-v2-design']);

// populate form from previous design
this.forms.setValue(previousDesign)
```